docs/_articles/en/v3.x/salesforce-graph-engine/rules.md

```diff
@@ -71,6 +71,39 @@ The `stripInaccessible` warning is thrown for all `stripInaccessible` checks on
 
 Graph Engine ran into an error while walking this path. Manually verify that you have a sanitizer on the path, and add an engine directive to skip the path. Next, create a GitHub issue for the Code Analyzer team that includes the error and stack trace so we can research and resolve it. After we determine a fix for the issue, check Code Analyzer [Release Information](https://forcedotcom.github.io/sfdx-scanner/en/v3.x/release-information/) for more info.
 
+## UnusedMethodsRule
```

Comment:
I missed this the first time around. The rule's name is actually "UnusedMethodRule", not "UnusedMethodsRule". That should be changed here and elsewhere.